app/web/features/search/SearchPage.tsx

```diff
@@ -112,18 +119,13 @@ export default function SearchPage({
       completeProfileFilter,
     ],
     ({ pageParam }) => {
-      // @ts-ignore @TODO David fixing these in a separate PR
-      const hostingStatusFilterComparation = parseInt(hostingStatusFilter);
-
       return service.search.userSearch(
         {
           query: queryName,
           bbox: locationResult.bbox,
           lastActive: lastActiveFilter === 0 ? undefined : lastActiveFilter,
           hostingStatusOptions:
-            hostingStatusFilterComparation === 0
-              ? undefined
-              : [hostingStatusFilter],
+            hostingStatusFilter.length === 0 ? undefined : hostingStatusFilter,
```

Comment:
Since you have an empty array as the default state in the `useState` above, shouldn't the `hostingStatusFilter.length === 0` case also be an empty array (rather than `undefined`)? Either way they should match the default states.

Reply:
ah I got what you mean, well basically everything that is `undefined` isn't sent to the server, so basically following that rule (this rule applies for this function, isn't something standard in our code) when is empty basically don't send anything to the server, as we were doing before.

Reply:
like maybe sending an empty array to the server is the same as undefined, but I rather don't risk to do it, also maybe if the backends changes (by mistake for example) will the frontend in that scenario.  
If we don't send anything (like before) then we make it more retro-compatible

Reply:
basically I debug what we sent to the server, so my changes don't affect and we don't send any extra info

Reply:
Ah sorry you're right I thought this was the value for `hostingStatusFilter` but it's actually the value for `hostingStatusOptions`